cmd/gateway/main_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Main", func() {
```

Comment:
I want to double check if going forward we're ok with using ginko for simple unit tests. 

So far, we've been using ginkgo for testing  public interfaces of types, usually with complex behavior with multiple cases, with preference for go unit test style tests for testing functions.

Reply:
Ok, I wasn't sure what the preferences were since I saw a mixture of both. I'm just used to using ginkgo which is why I used it here, but I'm open to any patterns you all have established.

Reply:
I think we stick with our pattern of using go unit style tests + gomega when testing simple functions. It's kind of a confusing pattern though and we need to document it somewhere.